src/validations/module.mk

```
@@ -1,5 +1,5 @@
 # Variables
-OSCAL_CLI = npx oscal@2.0.5
+OSCAL_CLI = npx oscal@next
```

Blocking: I accidentally took care of this in #901 per https://github.com/GSA/fedramp-automation/pull/901#issuecomment-2474788805. That was unintentional and I will have to lock myself down to prevent such mistakes as well, but I want this pinned. We will need to determine a matrix strategy in CI/CD to try the current pinned release and `oscal@next` in parallel to do some early warning testing as you are aiming for here.

```suggestion
OSCAL_CLI = npx oscal@2.0.6
```